build.gradle

```diff
@@ -186,14 +201,14 @@ dependencies {
     implementation 'com.github.LBNL-UCB-STI:helics-wrapper:v3.3.0'
 
     // GPLv3
-    implementation group: 'org.matsim.contrib', name: 'multimodal', version: '0.10.0'
-    implementation group: 'org.matsim.contrib', name: 'bicycle', version: '0.10.0'
+    implementation group: 'org.matsim.contrib', name: 'multimodal', version: '12.0'
+    implementation group: 'org.matsim.contrib', name: 'bicycle', version: '12.0'
 
-    implementation(group: 'org.matsim.contrib', name: 'decongestion', version: '0.11.0-2018w44') {
+    implementation(group: 'org.matsim.contrib', name: 'decongestion', version: '12.0') {
         exclude group: 'org.matsim', module: 'matsim'
     }
 
-    implementation(group: 'com.github.wrashid.matsim', name: 'matsim', version: '0.10.1-beam-15') {
+    implementation(group: 'com.github.LBNL-UCB-STI.matsim', name: 'matsim', version: '12.0-beam-5') {
```

Comment:
This is from branch [beam-patches-12.x](https://github.com/LBNL-UCB-STI/matsim/tree/beam-patches-12.x)